WordPress/Classes/Extensions/Colors and Styles/UIColor+MurielColors.swift

```diff
@@ -276,6 +276,15 @@ extension UIColor {
         return .neutral(.shade30)
     }
 
+    /// For now, just for the retry view background
+    static var retryBackground: UIColor {
```

Comment:
I added this color from https://github.com/wordpress-mobile/WordPress-iOS/issues/12517#issuecomment-532935835 - is there a reference name for this color?

Reply:
Hi @frosty, just wanted to ask for your input here. The [design](https://github.com/wordpress-mobile/WordPress-iOS/issues/12517#issuecomment-532935835) calls for a "Gray-10 / systemGray6 (iOS 13)" color background for the retry view, and I see we don't have that combination in code. I temporarily added it here, but wondering if you could guide me on how to handle this. I'm also thinking these colors already have semantic names defined somewhere but I'm not sure where to find them.

Reply:
Hey, thanks for the ping! I think it's okay to add this here – as you say, it doesn't match our existing colors so this looks fine 👍 Thanks for checking. Could we maybe clarify the name + comment slightly by changing it to `imageViewRetryBackground` or something similar?

Reply:
Thanks for the reply, @frosty! Will update the name and comment.